src/canvas/Paths/FreeLinePath.tsx

```diff
@@ -1,25 +1,7 @@
-import { PatternPath } from './PatternPath';
-import { CurveFitter } from '../Geometry/CurveFitter';
+import { Path } from './Path';
 import { Point } from '../Geometry/Point';
-import { Segment } from '../Geometry/Segment';
 
-export class FreeLinePath extends PatternPath { 
-    protected _createPathsFromSegments = (segments: Segment[]): PatternPath[] => {
-        const paths: FreeLinePath[] = [];
-        segments.forEach(segment => {
-            paths.push(new FreeLinePath(this._type, segment));
-        });
-
-        return paths;
-    };
-
-    protected _setFittedSegment = (): void => {
-        this._path2D = new Path2D();
-        this._isPath2DValid = true;
-        this._fittedSegment = CurveFitter.Fit(this._points);
-        this._fittedSegment.draw(this._path2D);
-    };
-    
+export class FreeLinePath extends Path { 
     protected _addPoint = (point: Point): void => {
         this._points.push(point);
```

Comment:
We should probably not add duplicate points.

Reply:
I added a check using the .equals method of Point class. It seems to work well even with the epsilon that's there.

Reply:
Great!